components/theme/footer.module.scss

```diff
@@ -0,0 +1,45 @@
+@use "/styles/variables" as variables;
+
+.footer {
+    min-height: 300px;
+
+    background-color: variables.$primary-b;
+
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    text-align: center;
+    justify-content: center;
+    
+    color: white;
+
+    .title {
```

Comment:
Apply global styling to these element after changing the styling into classes or mixins